packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_kokkos_def.hpp

```diff
@@ -312,74 +312,86 @@ namespace MueLu {
 
     // build the dof-based column map containing the local dof ids belonging to blkSize rows in matrix
     // the DofIds may not be sorted.
```

Comment:
I guess this comment is no longer valid?

Reply:
Yep, forgot to update the comments. Have updated now